planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/goal_planner_module.cpp

```diff
@@ -2766,10 +2779,15 @@ PathDecisionState PathDecisionStateController::get_next_state(
   // if object recognition for path collision check is enabled, transition to DECIDING to check
   // collision for a certain period of time. Otherwise, transition to DECIDED directly.
   if (parameters.use_object_recognition) {
-    next_state.state = PathDecisionState::DecisionKind::DECIDED;
```

Comment:
In prior PR, I mistakenly changed the next state to DECIDED. DECIDING is correct for this case.